Tests/NostrSDKTests/Bech32Tests.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  Bech32Tests.swift
+//
+//  Modified by Terry Yiu on 5/24/23.
+//  Created by Evolution Group Ltd on 12.02.2018.
+//  Copyright © 2018 Evolution Group Ltd. All rights reserved.
```

Comment:
Are you sure this is okay to copy in to our project with an MIT license? I didn't check. Just wanting to make sure we do due diligence.

Reply:
I sourced it from here and it's also MIT licensed. https://github.com/0xDEADP00L/Bech32

I should ask the folks from Nos, though, for some of the other code.

Reply:
Permission to redistribute this code under MIT license granted nostr:note1q39598qkdc093sdq4enudjf0dall76s7n779k07nutgd9r2zt6vq96l8c2